Formula/k/kanata-tray.rb

```diff
@@ -0,0 +1,32 @@
+class KanataTray < Formula
+  desc "System tray for kanata keyboard remapper"
+  homepage "https://github.com/rszyma/kanata-tray"
+  url "https://github.com/rszyma/kanata-tray/archive/refs/tags/v0.7.1.tar.gz"
+  sha256 "adeba8e8844fa9f5a051e98365e90ac8e0cb6661d61ac61627753582718d2062"
+  license "GPL-3.0-or-later"
+  head "https://github.com/rszyma/kanata-tray.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    # Requires CGO for system tray integration
+    ENV["CGO_ENABLED"] = "1"
+
+    commit = build.head? ? Utils.git_short_head : "9af15f4"
+    build_date = time.iso8601
+
+    ldflags = %W[
+      -s
+      -w
+      -X main.buildVersion=#{version}
+      -X main.buildHash=#{commit}
+      -X main.buildDate=#{build_date}
+    ]
+
+    system "go", "build", *std_go_args(ldflags:, output: bin/"kanata-tray")
```

Comment:
I think this is a default value.
```suggestion
    system "go", "build", *std_go_args(ldflags:)
```